src/fromager/commands/download_sequence.py

```diff
@@ -23,13 +24,19 @@
     default=False,
     is_flag=True,
 )
+@click.option(
+    "--num-threads",
+    default=3,
```

Comment:
Why not `os.cpu_count`?

Reply:
I'm worried about throttling on the server side causing errors. Maybe that's an overblown concern for modern services?

Reply:
Oh yeah, we might get rate limited as well. make sense

Reply:
I went ahead and set it to the CPU count. Let's see what that does in the build environment.